app/src/main/java/ch/hikemate/app/ui/map/RunHikeScreen.kt

```diff
@@ -263,30 +256,41 @@ fun RunHikeBottomSheet(
 
             DetailRow(
                 label = stringResource(R.string.run_hike_screen_label_current_elevation),
-                value = "50m")
+                // TODO hardcoded to 50m for now
+                value =
+                    stringResource(R.string.run_hike_screen_value_format_current_elevation)
+                        .format(50))
```

Comment:
Nitpick: you actually don't need to call `format` manually, you can just pass the arguments directly to `stringResource`, it calls `format` under the hood.

```suggestion
                    stringResource(R.string.run_hike_screen_value_format_current_elevation, 50))
```

Reply:
done